wazneinfo.md

```diff
@@ -54,4 +54,5 @@ Rzeczy tak wielkie jak sam PashaBiceps sprawdź mnie\
 Bo kiedy ja wchodzę to lepiej\
 Nie stawać tu na mojej drodze\
 Nie mogę tak dłużej i robię ten reload\
-I strzelam z akacza i kolejny zginął
\ No newline at end of file
+I strzelam z akacza i kolejny zginął
+hdjshdjshd smdnjkgdsgfef
```

Comment:
To się wymawia `hjdshjdshd` a nie `hdjshdjshd`. Proszę natychmiast poprawić

Reply:
nie!!!1! buuu